requirements/base.txt

```diff
@@ -16,7 +16,7 @@ redis==5.0.4
 
 # Inherited a4-core requirements
 Django==4.2.13
-django-allauth==0.55.0
+django-allauth==0.63.2
```

Comment:
should this be `django-allauth[socialaccount]` as described here https://docs.allauth.org/en/latest/release-notes/recent.html#backwards-incompatible-changes ?